test/manual/cssPrefix.html

```diff
@@ -6,7 +6,7 @@
 <style>
 #el {font-size:20pt}
 </style>
-<script src="../../bower_components/jquery/jquery.js"></script>
+<script src="https://code.jquery.com/jquery-1.7.2.min.js"></script>
```

Comment:
Trying to figure out install multiple version of same package as dependency, but actually there's no appropriate solution for that.
- Ref: https://github.com/npm/npm/issues/5499

Having a copy of legacy jQuery at `unit/lib` seems to be best for now.